Airports/EGJJ/Sector.txt

```diff
@@ -72,6 +72,7 @@ COORD:N050.00.00.000:W002.29.35.000
 ;NORTHBIT CTR ORTAC/ LON S line
 SECTORLINE:JJCTR-N-ORTAC-LONS
 DISPLAY:JJCTR-N-ORTAC:JJCTR-N-ORTAC:London AC Worthing
+DISPLAY:London AC Worthing:London AC Worthing:JJCTR-N-ORTAC
```

Comment:
I think there should be some more lines here, i.e S21, S22 and S20 should get the line, when JJCTR-N-ORTAC or S6 is covering the Jersey Zone

Reply:
> I think there should be some more lines here, i.e S21, S22 and S20 should get the line, when JJCTR-N-ORTAC or S6 is covering the Jersey Zone

👍 Agreed - we're looking to phase out use of 'London AC Worthing'